pkg/operator/ceph/object/admin.go

```diff
@@ -49,6 +49,18 @@ func runAdminCommandNoRealm(c *Context, args ...string) (string, error) {
 	return output, nil
 }
 
+func RunAdminCommandNoRealm(c *Context, args ...string) (string, error) {
```

Comment:
Why not just exporting the existing `runAdminCommandNoRealm` instead of duplicating?